### PR TITLE
[Alamofire 5]

### PR DESCRIPTION
**Pull request by MoElnaggar14**
- [x] update Alamofire version
- [x] update swift version

### CODE REVIEW

File: SwiftCairo-App/Networking/Response/ServerResponse.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+//  SwiftCairo-App
+//
+//  Created by Mohammed Elnaggar on 2/22/20.
+//  Copyright © 2020 abdelrahman mohamed. All rights reserved.
+//
+
+import Foundation
+
+enum ServerResponse<T> {
+    case success(T), failure(Error)
```
Comment (Yoloabdo) [inline SwiftCairo-App/Networking/Response/ServerResponse.swift]: Why do we need this? since there's Result now so no need for it

**Review by Yoloabdo — CHANGES_REQUESTED** [review]
Just one comment, thanks a lot

File: SwiftCairo-App/Networking/Response/ServerResponse.swift (hunk 1 above)
Comment (MoElnaggar14) [reply SwiftCairo-App/Networking/Response/ServerResponse.swift]: yeah, I'll replace it with AFResult

**Review by Yoloabdo — APPROVED** [review]

**Comment by Yoloabdo** [comment]
Looks good, merged! thanks a lot!

**Pull request merged**